tools/autograd/gen_variable_type.py

```diff
@@ -601,6 +675,21 @@ def wrap_view_single(output_var, base_var):
             return 'as_variable({})'.format(call), []
 
     def emit_call(env):
+        pre_call_block = ''
```

Comment:
can we make this its own function?  can be something that returns a pre_call, post_call tuple, including the NDEBUG.